pkg/querier/queryrange/query_range.go

```diff
@@ -380,6 +395,46 @@ func (s *SampleStream) MarshalJSON() ([]byte, error) {
 	return json.Marshal(stream)
 }
 
+// statsMerge merge the stats from 2 responses
```

Comment:
It can merge more than 2 responses right?

Reply:
done